### ISSUE TITLE
Usability: NVDA is not announcing the name of the editor groups.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17231]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

Steps to Reproduce:
Prerequisite: NVDA need to be opened.
1.Open Visual studio code editor.
2.Using "Ctrl+1" key navigate to the left editor.
3.Using "Ctrl+2" key navigate to the side editor. 
4.Using "Ctrl+3",key navigate to the Right editor.

**Expected Result:**
NVDA should announce the name of the editor group  while navigating to 
Ex: NVDA should announce "app.js text editor 1" when the user is navigated to first editor.

**Actual Result:**
NVDA is not reciting the name for the editor groups while navigating to it.

User Impact:
Screen Reader user doesn't get to know the which Editor they are in and on pressing "Ctrl+1" they are navigated to the editor or not is not known.

![inspectinformation_visualstudiocodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212700/289d2b04-c2b5-11e6-9680-3f2e5cfd4bcb.jpg)
![narratorbuddyinformation_vscodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212701/298d642a-c2b5-11e6-9ecc-6e92fc0d0f9d.jpg)
![nvda_speechviewerinformation_vscodeeditor_editor](https://cloud.githubusercontent.com/assets/22956130/21212702/2c221a3c-c2b5-11e6-8a63-01d97de999c8.jpg)

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/17230
